lib/Core/Executor.cpp

```diff
@@ -524,14 +524,18 @@ void Executor::initializeGlobals(ExecutionState &state) {
     globalAddresses.insert(std::make_pair(f, addr));
   }
 
+#ifndef WINDOWS
```

Comment:
Does it make sense to worry about Windows?

Reply:
Good question, we still have some windows code distributed over the codebase.
`errno` is a bit different for them. Depends on what we would like to support.

Reply:
OK, let's leave it like this for now, we actually had similar `ifdef`s before.